KyberAdapter/contracts/interfaces/IAdapter.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IAdapter {
+    function name() external view returns (string memory);
+
+    function swapGasEstimate() external view returns (uint256);
+
+    function swap(
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.htm